index.html

```diff
@@ -1338,11 +1338,12 @@ <h4>Listing</h4>
                                         with the URL of the next subset.
                                     </span>
                                     <span class="rfc2119-assertion" id="tdd-reg-list-pagination-header-nextlink-attr">
-                                        The `next` link MUST have the same `limit` argument given on the initial
-                                        request as well as a zero-based `offset` argument anchored at
-                                        the beginning of the next subset.
+                                        The `next` link MUST include all arguments needed to produce the same set of
+					data and its ordering. Particularly, it MUST include the same `limit` argument
+					given on the initial request as well as a zero-based `offset` argument anchored
+					at the beginning of the next subset.
                                     </span>
-                                    The link may be absolute or relative to directory API's base URL.
+                                    The link MAY be absolute or relative to directory API's base URL.
```

Comment:
To make this an assertion, it should be added to an assertion span, e.g.:
```
<span class="rfc2119-assertion" id="tdd-reg-list-pagination-header-nextlink-base">
This link MAY be ...
</snap>
```

But I suggest changing it to a MUST. IMO, "MAY" followed by "or" doesn't really prescribe anything formally.

Reply:
I agree with Farshid's rewording, using MUST.  Also, need the span tag (with a unique id, please).